src/main/java/de/hysky/skyblocker/utils/Utils.java

```diff
@@ -190,18 +185,14 @@ private static void onLeaveSkyblock() {
         }
     }
 
-    public static String getLocation() {
+    public static String getLocationScoreboard() {
```

Comment:
This name is confusing and isn't in line with what hypixel calls it. The method should be renamed to `getArea` or `getIslandArea`